File: LanguAI.Frontend/src/api/models/message-view-model.ts
/* tslint:disable */
/* eslint-disable */
import { MessageStatusEnum } from '../models/message-status-enum';
export interface MessageViewModel {
  id?: number | null;
  recipientId?: number;
  senderId?: number;
  sentAt?: string;
  status?: MessageStatusEnum;
  text?: string | null;
}

File: LanguAI.Backend/LanguAI.Backend/ViewModels/Message/MessageViewModel.cs
using LanguAI.Backend.Core.Enums;

namespace LanguAI.Backend.ViewModels.Message;

public class MessageViewModel
{
    public int? Id { get; set; }

    public string Text { get; set; }

    public int SenderId { get; set; }

    public int ReceiverId { get; set; }

    public DateTime SentAt { get; set; }

    public MessageStatusEnum Status { get; set; }
}
